modules/client/src/allowlistVoting/interfaces.ts

```diff
@@ -10,23 +8,23 @@ export interface IAllowlistVotingContextPluginState extends IAllowlistVoting {}
 
 export interface ProposalAction {
   to: string;
-  value: BigNumberish;
+  value: BigInt;
   data: Uint8Array;
 }
 
 export interface Proposal {
-  id: BigNumberish;
+  id: number;
   open: boolean;
   executed: boolean;
-  startDate: BigNumberish;
-  endDate: BigNumberish;
-  snapshotBlock: BigNumberish;
-  supportRequired: BigNumberish;
-  participationRequired: BigNumberish;
-  votingPower: BigNumberish;
-  yes: BigNumberish;
-  no: BigNumberish;
-  abstain: BigNumberish;
+  startDate: number;
+  endDate: number;
+  snapshotBlock: BigInt;
+  supportRequired: BigInt;
+  participationRequired: BigInt;
+  votingPower: BigInt;
+  yes: BigInt;
+  no: BigInt;
+  abstain: BigInt;
```

Comment:
It might be the exact same, but technically the TypeScript type name for them is `bigint` lowercase. `BigInt(...)`  is the constructor, although it may pass

Reply:
switched